docs/components/identity/user.md

```diff
@@ -0,0 +1,58 @@
+---
+id: user
+title: User management
+description: "Learn how to manage users in your Self-Managed Orchestration cluster."
+---
+
+:::note
+User management in Identity is only needed by Self-Managed deployments. For SaaS, see how to [connect your identity provider](http://localhost:3000/docs/next/components/console/manage-organization/external-sso/).
+:::
+
+Identity provides unified, cluster-level identity management and authorizations. User management is a key part of identity management, and allows you to create, update, and delete users in your Orchestration cluster.
```

Comment:
```suggestion
User management is a key part of identity management, and allows you to create, update, and delete users in your Orchestration cluster.
```

I think here I'd also remove the Identity definition at the start, this page is about user management which the sentence after captures well

Reply:
Though looking at this and the next section `User management overview`, are they both needed?

Reply:
They are similar, I'll merge the sections